Add PureScript icon

i-am-the-slime opened the pull request:
Closes #602

orta commented:
That SVG is too small, it will probably end up looking off compared to the rest - the README has all the sizing info

i-am-the-slime commented:
Sorry, no idea how this happened, I guess I got confused because `gulp svg` also changed `cake.svg` and `cu.svg` for me and I reverted something wrong.

orta commented:
Great 👍🏻
